### PR TITLE
skip /art-appraisals

### PR DESCRIPTION
**Pull request by pvinis**
context: https://artsy.slack.com/archives/C02BAQ5K7/p1654740098225179

### CODE REVIEW

File: apple-app-site-association.json [hunk 1]
```diff
@@ -17,7 +17,9 @@
       {
         "appID": "23KMWZ572J.net.artsy.artsy",
         "paths": [
+          "NOT /#",
```
Comment (pvinis) [inline apple-app-site-association.json:20]: obey alphabet

File: apple-app-site-association.json [hunk 2]
```diff
@@ -17,7 +17,9 @@
...
           "NOT /2016-year-in-art",
+          "NOT /art-appraisals",
```
Comment (pvinis) [inline apple-app-site-association.json:22]: https://artsy.slack.com/archives/C02BAQ5K7/p1654740098225179

File: apple-app-site-association.json [hunk 3]
```diff
@@ -44,4 +45,4 @@
       }
     ]
   }
-}
\ No newline at end of file
+}
```
Comment (pvinis) [inline apple-app-site-association.json:48]: newline babyyyy

**Comment by pvinis** [comment]
peril freaked out?

**Comment by pvinis** [comment]
closing

**Comment by damassi** [comment]
🙏 @pvinis for hopping on this so fast

**Comment by artsyit** [comment]
:rocket: PR was released in [`v1.5.2`](https://github.com/artsy/artsy-eigen-web-association/releases/tag/v1.5.2) :rocket:

**Pull request merged**